docs/source/Tutorial.md

```diff
@@ -1542,11 +1542,6 @@ Note how we create `Vec3` struct in-line in the table. Unlike tables, structs
 are simple combinations of scalars that are always stored inline, just like
 scalars themselves.
 
-**Important**: Unlike structs, you should not nest tables or other objects,
```

Comment:
This comment was for all languages, and the new ones are C/C++ only. The comment applies to all languages.

Reply:
Ok, seems like it is indeed a little more complicated. I looked through all the other languages. If I understood correctly, this comment is actually well-placed fro all languages except C, C++ and Rust. Because these languages has a code snippet before the original comment, which consists only of one call to a builder-function. This call obviously cannot be nested/interlaced with other calls. While other languages have examples at this point, where several calls are done on a builder object, thus the comment makes sense. Yet mentioning of "start" and "end" is still confusing. Is this correct?

Reply:
Look through the examples above, almost all languages have a way of building a table that starts with a builder, then individual calls to create fields, then an End call. This API is susceptible to nesting problem the comment is referring to. As opposed to Create() calls (where each field is an argument) that enforce proper nesting since you can't construct anything "in between" calls there.